[branch-0.17.0-0.3] Deletion of AWS, AKS and GCP references (#682)
Stratio/kind: stratio-docs/en/modules/operations-manual/pages/operations-manual.adoc

== Generation of custom images

> For the generation of custom images in AWS manages, the next xref:operations-manual:image-builder/aws-image-builder.adoc[reference documentation] has been created.

Author: ¿Es correcto que dejemos solo esta página del _image builder_?

Reviewer: No deberíamos tener ninguna referencia de image-builder

...
----

> NOTE: Unmanaged providers will have one _MachineHealthCheck_ for the _control-plane_ and another for the _worker_ nodes, while managed ones (EKS) will only have the second one.

Author: ```suggestion
NOTE: EKS will have a _MachineHealthCheck_ for the _worker_ nodes.
```
